modules/comment/comment.service.js

```diff
@@ -63,11 +78,11 @@ class CommentsService {
     const newRate = rateSum / rateCount;
 
     const newUserRates = rate
-      ? userRates.map(item =>
-          String(item.user) === String(user._id)
+      ? userRates.map(item => {
+          return String(item.user) === String(user._id)
```

Comment:
Remove this useless `return`, cause we use here arrow function

Reply:
fixed